### ISSUE TITLE
New translation: Thai

### ISSUE THREAD

**Issue by Mujina7569** [squidfunk/mkdocs-material#1340]
``` jinja
{% macro t(key) %}{{ {
  "language": "th",
  "clipboard.copy": "คัดลอก", 
  "clipboard.copied": "คัดลอกแล้ว", 
  "edit.link.title": "แก้ไขหน้านี้", 
  "footer.previous": "ก่อนหน้า", 
  "footer.next": "ต่อไป", 
  "meta.comments": "ความคิดเห็น", 
  "meta.source": "แหล่งที่มา", 
  "search.language": "th",
  "search.placeholder": "ค้นหา", 
  "search.result.placeholder": "พิมพ์เพื่อเริ่มค้นหา", 
  "search.result.none": "ไม่พบเอกสารที่ตรงกัน", 
  "search.result.one": "พบเอกสารที่ตรงกัน", 
  "search.result.other": "พบ # เอกสารที่ตรงกัน", 
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "ข้ามไปที่เนื้อหา", 
  "source.link.title": "ไปที่ Repository", 
  "toc.title": "สารบัญ" 
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Pretty cool, thanks! Will integrate it into the next release.

**Comment by squidfunk** [comment]
Google Translate tells me that there are certain translations for "Repository" - does it make sense to translate this as well?

- กรุ _dungeon, repository, hiding place, chamber, underground hiding place, cell_
- ที่รองรับที่ใส่ _repository_
- ที่รับ _repository_

**Comment by Mujina7569** [comment]
> Google Translate tells me that there are certain translations for "Repository" - does it make sense to translate this as well?
> 
> * กรุ _dungeon, repository, hiding place, chamber, underground hiding place, cell_
> * ที่รองรับที่ใส่ _repository_
> * ที่รับ _repository_

No... It's make more confuse and i saw they use *Repository* instead

**Comment by squidfunk** [comment]
Released as part of 4.5.1.
